ios/TwakeMailNSE/NotificationService.swift

```diff
@@ -143,6 +145,7 @@ class NotificationService: UNNotificationServiceExtension {
         self.modifiedContent?.body = message
         self.modifiedContent?.badge = NSNumber(value: 1)
         self.modifiedContent?.sound = .default
+        self.modifiedContent?.userInfo = ["data": originalContent.userInfo]
```

Comment:
why do we need `data` key?

Reply:
`onMessage` of FCM Flutter is using data key for its payload.